### PR TITLE
feat: Add preferredAudioLabel to PlayerConfiguration

### PR DESCRIPTION
**Pull request by tyrelltle**
Closes [#4762](https://github.com/shaka-project/shaka-player/issues/4762).

The change is to add PreferredAudioLabel to player configuration so that host apps can decide the default variant label to choose at very beginning.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -90,4 +91,4 @@ Wayne Morgan <wayne.morgan.dev@gmail.com>
 Raymond Cheng <raycheng100@gmail.com>
 Blue Billywig <*@bluebillywig.com>
 João Nabais <jlnabais@gmail.com>
-Koen Romers <koenromers@gmail.com>
+Koen Romers <koenromers@gmail.com>
```
Comment (avelad) [inline AUTHORS:94]: Please revert this final file line

File: CONTRIBUTORS [hunk 2]
```diff
@@ -130,4 +131,4 @@ Yohann Connell <robinconnell@google.com>
 Raymond Cheng <raycheng100@gmail.com>
 Janroel Koppen <j.koppen@bluebillywig.com>
 João Nabais <jlnabais@gmail.com>
-Koen Romers <koenromers@gmail.com>
+Koen Romers <koenromers@gmail.com>
```
Comment (avelad) [inline CONTRIBUTORS:134]: Please revert this final file line

File: demo/config.js [hunk 3]
```diff
@@ -477,6 +477,7 @@ shakaDemo.Config = class {
 
     this.addSection_(MessageIds.LANGUAGE_SECTION_HEADER, docLink)
         .addTextInput_(MessageIds.AUDIO_LANGUAGE, 'preferredAudioLanguage')
+        .addTextInput_(MessageIds.AUDIO_LANGUAGE, 'preferredAudioLabel')
```
Comment (avelad) [inline demo/config.js:480]: It is necessary to add a different message since it does not do the same thing as preferredAudioLanguage

**Review by avelad — CHANGES_REQUESTED** [review]

**Comment by avelad** [comment]
Please, fix the lint errors. Thanks!

File: AUTHORS (hunk 1 above)
Comment (tyrelltle) [reply AUTHORS]: updated

**Comment by tyrelltle** [comment]
@avelad 
Thanks for the comments I have updated accordingly.

One question:
Its added to [v4.4](https://github.com/shaka-project/shaka-player/milestone/26), does it mean the change will not be added to current v4.2.x and v4.3.x ?

What makes a PR to be eligible for getting cherry picked into current release (4.2.x and 4.3.x) ?

Thanks.

**Review by avelad — APPROVED** [review]

**Review by theodab — APPROVED** [review]

**Pull request merged**